enhancements/testdata-separation.md

```diff
@@ -0,0 +1,109 @@
+---
+title: testdata-separation
+authors:
+  - "@theishshah"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-07-12
+last-updated: 2021-07-20
+status: TBD
+---
+
+# testdata-separation
+
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+The current bundle includes both an operator as well as its testdata. This results in inflated binaries and scenarios where users may want to release an operator bundle sans testdata but are unable to do so. The solution to this issue is to provide users the ability to use and generate a slim bundle which is decoupled from the testing necessities. Due to the nature of the testdata, it needs the rest of the bundle so it has limited value alone.
+
+## Motivation
+
+The motivation for this EP is to provide users with the option to deliver a slim bundle without testdata.
+* Creates a smaller size bundle to ship
+* Prevents including testing resources in shipped bundle
+  * Stop potential elevated RBAC's in the test bundle from being included in production environments
+
+### Goals
+
+The goal is to create a solution which allows developers to continue to access and utilize the testdata while still having the option to ship an operator without said testdata in the bundle.
+
+### Non-Goals
+
+The goal of this EP is not to change the contents of the bundle nor the testdata, but just to reorganize them, either with multiple bundles or by allowing the tools to modify a bundle to achieve the desired state.
+
+## Proposal
+
+Please note that this proposed solution was elevated from the Alternatives selection after discussions, and the previous implementation has been moved down.
+
+Hard Split
+* Split existing bundle in a testdata only bundle paired with a operator essentials only bundle
```

Comment:
When a project is initialized, the `Makefile` is scaffolded with the assumption that there is a single bundle per project (see [here](https://github.com/operator-framework/operator-sdk/blob/2f6e5242a0a367feca82891d76283c537ee8fedb/internal/plugins/manifests/v2/init.go#L160-L186)). How will we overcome this assumption?